### PR TITLE
Feat - Add StoryblokBridge Options

### PR DESCRIPTION
**Pull request by Edo-San**
The purpose of this PR is to let the user be able to pass `StoryblokBridge` options via `astro.config`.

The idea behind it is:
- `resolvedOptions.bridge` should be a `boolean` or an instance of `StoryblokBridgeConfigV2` options
- If a user provides a truthy value, the options are passed to the StoryblokBridge constructor. This means that providing `bridge: true` would behave the same as providing `bridge: { foo: "bar" }`
 
Initially, I tried to cherry-pick only the keys that are part of the `StoryblokBridgeConfigV2` interface from the provided object, to avoid injecting useless (/ potentially harmful) config options to the bridge (i.e. if you provide this object:
```
bridge: { foo: "bar", preventClicks: true }
```
all keys will be merged with the bridge configuration).

In the end, I noticed that there's actually no harm in doing that, plus `JSON.stringify` prevents `eval`-like code injections so I just kept it super straightforward 😄 (And AFAIK there's no easy way to create an array of keys from an interface..).

I also added `basicSsl` to `playground-test` config to be able to try it out in the Storyblok preview, but I commented it out for the sake of existing e2e tests and their configuration which points to `http://localhost:3000`.

As for testing, I tried to set up an e2e test to check for the `preventClicks` option but this behaviour depends heavily on the Storyblok Bridge implementation. Namely, the page has to be in an iframe in order to be able to receive postMessages, plus there's quite some logic in the [Bridge init](https://github.com/storyblok/storyblok-bridge/blob/6b9007507b0ad549fc9d5cfaf673d27d290edbe3/src/bridge-private.js#L169) that's pretty hard to mock. I tested it manually from the Storyblok editor directly and it worked out.

### CODE REVIEW

**Review by alexjoverm — CHANGES_REQUESTED** [review]
Neaaaat @Edo-San it surprised me as I thought we had this in place in Astro SDK, but definitely not as in others SDKs config is passed on `useStoryblokBridge` call, but Astro SDK "hardcodes" that functionality, so makes total sense to pass it at init time 👌 

Can you shortly document this options in the Readme? We have [a section for the brigde](https://github.com/storyblok/storyblok-astro#using-the-storyblok-bridge), but feel free to do it as you consider the best!

File: lib/index.ts [hunk 1]
```diff
@@ -50,9 +58,9 @@ export type IntegrationOptions = {
    */
   apiOptions?: ISbConfig;
   /**
-   * A boolean to enable/disable the Storyblok JavaScript Bridge. Enabled by default.
+   * A boolean to enable/disable the Storyblok JavaScript Bridge or a StoryblokBridgeConfigV2 configuration object. Enabled by default.
```
Comment (manuelschroederdev) [inline lib/index.ts:61]: Maybe a bit clearer: ... or _provide_ a StoryblokBridgeConfigV2 ...
What do you think?

File: lib/index.ts [hunk 2]
```diff
@@ -8,7 +8,12 @@ import {
 } from "@storyblok/js";
 
 import type { AstroIntegration } from "astro";
-import type { ISbConfig, ISbRichtext, SbRichTextOptions } from "./types";
+import {
```
Comment (manuelschroederdev) [inline lib/index.ts:11]: Since it's just types can we keep the `import type`?

**Review by manuelschroederdev — COMMENTED** [review]
Love it, thanks a lot!
Just added two nitpicky comments. 😄
Let me know if you're short on time, I could also take care of the readme changes in that case.

File: lib/index.ts (hunk 1 above)
Comment (Edo-San) [reply lib/index.ts]: Sounds good. Updated!

**Comment by Edo-San** [comment]
> Neaaaat @Edo-San it surprised me as I thought we had this in place in Astro SDK, but definitely not as in others SDKs config is passed on `useStoryblokBridge` call, but Astro SDK "hardcodes" that functionality, so makes total sense to pass it at init time 👌
> 
> Can you shortly document this options in the Readme? We have [a section for the brigde](https://github.com/storyblok/storyblok-astro#using-the-storyblok-bridge), but feel free to do it as you consider the best!

I added a paragraph to the Readme under the bridge section. FYI there's also a `language` param that is not documented anywhere, apparently. I decided not to include that in the Readme because it's not even mentioned in this [guide](https://www.storyblok.com/docs/guide/in-depth/storyblok-latest-js-v2) 🤔 

Please let me know if everything is ok for you with this Readme integration @alexjoverm @manuelschroederdev 🙌

**Comment by manuelschroederdev** [comment]
> I added a paragraph to the Readme under the bridge section. FYI there's also a language param that is not documented anywhere, apparently. I decided not to include that in the Readme because it's not even mentioned in this [guide](https://www.storyblok.com/docs/guide/in-depth/storyblok-latest-js-v2) 🤔

Thank you. I think we can probably ignore the language parameter.

The added paragraph is perfect, but I'm unsure if and how we should document `resolveRelations`. In all other SDKs, you would usually make the API call and initialize the bridge in the same file(s), making it possible to resolve different relations depending on which you actually need. Here, it would just be possible to resolve relations globally.

**Comment by alexjoverm** [comment]
I think resolving relations in the bridge is irrelevant for Astro - what bridge does in Astro SDK is reload when you click save (no real-time editing), so the proper API calls will be perform after reloading. We're good

**Review by alexjoverm — APPROVED** [review]
Amazing PR, thanks @Edo-San !

**Comment by Edo-San** [comment]
> I think resolving relations in the bridge is irrelevant for Astro - what bridge does in Astro SDK is reload when you click save (no real-time editing), so the proper API calls will be perform after reloading. We're good

Should I just mention `preventClicks`? WDYT? 🤔

**Comment by manuelschroederdev** [comment]
Maybe let's mention `resolveRelations` but add a short disclaimer that it won't have any effect due to the way the bridge is utilized in the Astro SDK?

**Comment by Edo-San** [comment]
> Maybe let's mention `resolveRelations` but add a short disclaimer that it won't have any effect due to the way the bridge is utilized in the Astro SDK?

Makes sense. Updated 😉

**Pull request merged**